src/jetpack-editor-setup.js

```diff
@@ -26,6 +28,8 @@ const supportedJetpackBlocks = {
 	},
 };
 
+const JETPACK_LOCALE_DOMAIN = 'jetpack';
```

Comment:
The domain for Jetpack is `jetpack`, [here](https://github.com/Automattic/jetpack/blob/677b2f563a278bb581576bc938d15cffe0589933/projects/plugins/jetpack/extensions/blocks/contact-info/address/edit.native.js#L37) you can see an example.